# Goal
geometry_msgs/PoseStamped target_pose
---
# Result
bool success
---
# Feedback
float32 distance_to_goal
